pandas/plotting/_core.py

```diff
@@ -2616,45 +2660,12 @@ def hist(self, bins=10, **kwds):
         """
         return self(kind='hist', bins=bins, **kwds)
 
-    def kde(self, bw_method=None, ind=None, **kwds):
-        """
-        Kernel Density Estimate plot using Gaussian kernels.
-
-        In statistics, kernel density estimation (KDE) is a non-parametric way
-        to estimate the probability density function (PDF) of a random
-        variable. This function uses Gaussian kernels and includes automatic
-        bandwith determination.
-
-        Parameters
-        ----------
-        bw_method : str, scalar or callable, optional
-            The method used to calculate the estimator bandwidth. This can be
-            'scott', 'silverman', a scalar constant or a callable.
-            If None (default), 'scott' is used.
-            See :class:`scipy.stats.gaussian_kde` for more information.
-        ind : NumPy array or integer, optional
-            Evaluation points for the estimated PDF. If None (default),
-            1000 equally spaced points are used. If `ind` is a NumPy array, the
-            kde is evaluated at the points passed. If `ind` is an integer,
-            `ind` number of equally spaced points are used.
-        kwds : optional
-            Additional keyword arguments are documented in
-            :meth:`pandas.Series.plot`.
-
-        Returns
-        -------
-        axes : matplotlib.AxesSubplot or np.array of them
-
-        See also
-        --------
-        scipy.stats.gaussian_kde : Representation of a kernel-density
-            estimate using Gaussian kernels. This is the function used
-            internally to estimate the PDF.
-
-        Examples
-        --------
+    @Appender(_kde_docstring % {
+        'this-datatype': 'Series',
+        'sibling-datatype': 'DataFrame',
+        'examples': """
         Given a Series of points randomly sampled from an unknown
-        distribution, estimate this distribution using KDE with automatic
+        distribution, estimate its distribution using KDE with automatic
```

Comment:
I think the original was right: what we estimate is the distribution of the random sample. If you say "estimate its distribution" it looks like we are estimating the distribution of the distribution. But I'm not a native English speaker. What do you think?

Reply:
Saw your comment in my PR about this.

I personally prefer "this" because I feel it makes clearer that what is estimated is the unknown distribution. Otherwise "its" refers to the Series, which is early in the sentence, and is more confusing to me.

But I'm not a native English speaker. I'm fine with both options, don't like bikeshedding too much :)

Reply:
Me neither .. just because we are already thinking of it: changing the whole part to `estimate its PDF using KDE` might be an even better fit, as this is what we are really doing.

Reply:
I just changed it to that.